solution_files/coarse_set/cb40f3f7-ffe8-40e8-4544-f26aad5a8bd8_sol_30d863a7-ffde-40f9-8107-af771ec986b5.json

```diff
@@ -0,0 +1,174 @@
+            "run_time": {
+                "overall_time": {
+                    "seconds": 133699950.46700683
+                },
+                "preprocessing_time": {
+                    "seconds": 34.24659085273743
+                },
+                "algorithm_run_time": {
+                    "seconds": 133699916.22041598
+                },
+                "postprocessing_time": {
+                    "seconds": 0
+                }
+            },
+            "error_bound": 0.001,
+            "confidence_level": 0.99
+        },
+        {
+            "instance_data_object_uuid": "bae2da57-6a69-483e-95bc-b77f72ebfba8",
+            "run_time": {
+                "overall_time": {
+                    "seconds": 1822049254.3327522
+                },
+                "preprocessing_time": {
+                    "seconds": 109.89768028259277
+                },
+                "algorithm_run_time": {
+                    "seconds": 1822049144.435072
+                },
+                "postprocessing_time": {
+                    "seconds": 0
+                }
+            },
+            "error_bound": 0.001,
+            "confidence_level": 0.99
+        },
+        {
+            "instance_data_object_uuid": "72343006-774e-4192-b481-fa840ed25573",
+            "run_time": {
+                "overall_time": {
+                    "seconds": 133068577.38371417
+                },
+                "preprocessing_time": {
+                    "seconds": 21.355810165405273
+                },
+                "algorithm_run_time": {
+                    "seconds": 133068556.027904
+                },
+                "postprocessing_time": {
+                    "seconds": 0
+                }
+            },
+            "error_bound": 0.001,
+            "confidence_level": 0.99
+        },
+        {
+            "instance_data_object_uuid": "ea55abec-8253-445d-85fa-914948b5e5a5",
+            "run_time": {
+                "overall_time": {
+                    "seconds": 1827701433.7916307
+                },
+                "preprocessing_time": {
+                    "seconds": 112.39502286911011
+                },
+                "algorithm_run_time": {
+                    "seconds": 1827701321.3966079
+                },
+                "postprocessing_time": {
+                    "seconds": 0
+                }
+            },
+            "error_bound": 0.001,
+            "confidence_level": 0.99
+        }
+    ],
+    "compute_hardware_type": "quantum_computer",
+    "compute_details": {
+        "description": "Double factorized QPE resource estimates based on methodology of arXiv:2406.06335. Uses PyLIQTR logical resource estimates with BenchQ footprint analysis. Ground-state overlap assumed to be 0.8 and double-factorized truncation threshold to be 1e-3 Ha. Note that the truncation error is not included in the error bounds and that the SCF compute time is not included in the preprocessing time."
```

Comment:
Might be good to capture the classical hardware used for the pre-processing. It's not needed at this point, but would be nice to have